bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx60.html

```diff
@@ -0,0 +1,126 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_59" "firefox/mobile" "firefox/sendto" %}
+
+{% extends "firefox/whatsnew/base-pebbles.html" %}
+
+{% block page_title %}{{ _('See what’s new with Firefox') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_whatsnew_60' %}
+{% endblock %}
+
+{% set show_send_to_device = LANG not in settings.SEND_TO_DEVICE_LOCALES %}
```

Comment:
Remove `not`? (assume this is left over from debug 😄)

Reply:
Yyyyep, left over from ye olde debugging.